microsoft-edge/extensions-chromium/publish/api/using-addons-publish-api.md

```diff
@@ -0,0 +1,220 @@
+---
+description: REST endpoints to automate publishing updates to add-ons that are submitted to the Microsoft Edge Add-ons website.
+title: Using the Microsoft Edge Add-ons Publish API
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 08/10/2021
+ms.topic: conceptual
+ms.prod: microsoft-edge
+keywords: edge-chromium, extensions development, browser extensions, add-ons, partner center, developer, publish api, add-ons publish api
+---
+# Using the Microsoft Edge Add-ons Publish API
+
+<!-- find and resolve: tbd, html comments, https://addons.edge.microsoft.com/api -->
+
+This article, along with the [Add-ons Publish API Reference][AddonsPublishAPIRef], provides an overview of the proposed Microsoft Edge Add-ons Publish API.  We look forward to your suggestions and feedback on the proposed API contracts.  Please submit your feedback at (_link TBD_).
+
+The Microsoft Edge Add-ons Publish API provides a set of REST endpoints for programmatically publishing updates to add-ons submitted to the Microsoft Edge Add-ons website.  You can use these REST endpoints to automate the process of uploading and publishing add-ons into the Microsoft Edge Add-ons website.
+
+The tentative date for the availability of the APIs is December 2021.
+
+
+<!-- ====================================================================== -->
+## Terminology
+
+| Term | Definition |
+|---|---|
+| _product_ | A single product for which you have an entire entry at Partner Center. |
+| _product ID_ | The product ID of the product whose draft needs to be published.  This is the product ID of your product as shown at Partner Center. |
+| _submission_ | A version of the product.  Every update to a product is a submission, whether draft, in review, or published. |
+| _operation_ | A REST operation such as GET or PUT.  There are several operations, defined at [Add-ons Publish API Reference][AddonsPublishAPIRef]. |
+| _operation ID_ | The ID of a REST operation. |
+
+
+<!-- ====================================================================== -->
+## Before you begin
+
+To use the Microsoft Edge Add-ons Publish API, you need to enable the API for your project in the Microsoft Partner Center.
+
+1. Visit Microsoft Partner Center and login to the account from which you have already published an add-on.
+
+1. Under the **Microsoft Edge** program, a new page for **Publish APIs** appears.
+
+1. Enable the APIs.<!-- what UI action to do that?--> This step may take a few minutes.  After the APIs are enabled, the **Client ID**, **Client Secret**, and **Auth Token URL** are displayed on this page.
```

Comment:
Is it obvious at the page, how to enable the APIs?  What concrete UI step are we saying to do to accomplish this?  maybe:  Select the **Enable the APIs** button.  Add a screenshot here?

Reply:
@mikehoffms just yesterday there is a small change in the workflow. This should now be: **_Click the "Create API credentials" button to generate the API credentials._**